app/controllers/application_controller.rb

```diff
@@ -3,21 +3,25 @@
 class ApplicationController < ActionController::Base
   include Localization
 
-  helper_method :logged_in?, :current_user
+  helper_method :authenticated?, :current_user
 
   private
 
-  def logged_in?
+  def authenticated?
     session[:user_id].present?
   end
 
   def current_user
-    return unless logged_in?
+    return unless authenticated?
 
     @current_user ||= User.find_by(id: session[:user_id])
   end
 
-  def redirect_to_home
-    redirect_to root_path
+  def ensure_no_authentication
+    redirect_to root_path if authenticated?
+  end
+
+  def ensure_autentication
```

Comment:
typo, it should be `authentication` instead of `autentication`

Reply:
Fixed in fd1e8de4289edbd0b770bfaebcc777d7c0446fe5